recipes/ldas-tools-ldasgen/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ldas-tools-ldasgen" %}
+{% set version = "2.6.3" %}
+{% set sha256 = "e999263c2341a3935b5030c83304ce423da0aee2b3f719a86a172e5dad5b74ba" %}
+{% set ldas_tools_al_version = "2.6.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make
+  host:
+    - flex
+    - boost-cpp
+    - openssl
+    - zlib
+    - ldas-tools-al >={{ ldas_tools_al_version }}
+  run:
+    - ldas-tools-al >={{ ldas_tools_al_version }}
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/genericAPI  # [not win]
+    - test -f ${PREFIX}/lib/libStat${SHLIB_EXT}  # [not win]
+    - test -f ${PREFIX}/lib/libldasgen${SHLIB_EXT}  # [not win]
```

Comment:
Since you are skipping windows builds I don't think you need the selectors here on the test section, but I could be wrong.

Reply:
Aye, that's fair, but I put these in with an eye to future support of windows. If its cleaner to have them removed, that's fine.